### PR TITLE
Introduce DoctorGraphicalinterface

### PR DESCRIPTION
**Pull request by BoskiJanusz**
- New class to represent connection with doctor,
- QML to represent interface for doctor,
- register qml in main.cpp

### CODE REVIEW

File: modules/app/connections/DoctorGraphicalInterface.cpp [hunk 1]
```diff
@@ -0,0 +1,56 @@
+include "DoctorGraphicalInterface.hpp"
```
Comment (Aleksiiej) [inline modules/app/connections/DoctorGraphicalInterface.cpp:1]: # ma być

**Comment by ziobron** [comment]
Thanks @Aleksiiej for Code Review.
🏅 1 XP granted

**Review by Aleksiiej — APPROVED** [review]
was fine :)

**Comment by ziobron** [comment]
Thanks @Aleksiiej for Code Review.
🏅 1 XP granted

**Comment by ziobron** [comment]
Your PR was merged!
🏅 2 XP granted. Thanks @BoskiJanusz for making course materials better!

**Pull request merged**